ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -19,6 +19,10 @@
 
 package org.netbeans.modules.extbrowser;
 
+import com.sun.jna.platform.win32.Advapi32Util;
+import com.sun.jna.platform.win32.WinReg;
+import java.util.Locale
```

Comment:
missing ; at end of line.  You should try to compile using ant to get rid of the "basic compiler issue"